File: ejercicios/python/ejercicio066.py
def fibo_recursivo(termino):
   if (termino < 2):
      return termino
   else:
      return fibo_recursivo(termino - 1) + fibo_recursivo(termino - 2)

if __name__ == "__main__":
   numero = int(input("ingrese el numero : "))
   if (numero >= 0):
      print ("el termino " + str(numero) + " de la serie de fibonacci es " + str(fibo_recursivo(numero)))
   else:
      print("El numero debe ser positivo.")
      

File: ejercicios/c++/ejercicio066.cpp
#include <iostream>
using namespace std;

int fibo_recursivo(int termino){	if(termino < 2)		return termino;	else		return fibo_recursivo(termino-1) + fibo_recursivo(termino-2);}

int main() {	int numero;	cout << "ingrese un numero entero : ";	cin >> numero;	if (numero >= 0)		cout << "el termino " << numero << " de la serie de fibonacci es " << fibo_recursivo(numero) << endl;	else		cout << "El numero debe ser positivo." << endl;	return 0;}
